### PR TITLE
add jszip-utils@0.0.2 w/ npm autoupdate

### LINKED ISSUE #7819: [Request] Add jszip-utils
**Library name:** jszip-utils
**Git repository url:** https://github.com/Stuk/jszip-utils
**npm package url(optional):** 
**License(s):** JSZipUtils is dual-licensed. You may use it under the MIT license or the GPLv3 license. See LICENSE.markdown.
**Official homepage:** http://stuk.github.io/jszip-utils/
**Wanna say something? Leave message here:**

=====================
Notes from cdnjs maintainer:
You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by pvnr0082t**
PR for #7819 
@x09326 Please help me review this pull request, thank you.

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/Stuk/jszip-utils
 * Official website (optional, not the repository): https://stuk.github.io/jszip-utils
 * NPM package url (optional): https://www.npmjs.com/package/jszip-utils
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 33
   - Count of watchers: 6
   - Count of forks: 23
 * NPM download stats (optional):
   - Downloads in the last day: 13
   - Downloads in the last week: 58
   - Downloads in the last month: 319

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [ ] The lib has notable popularity
   * [ ] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

close #7819, cc @Stuk

### CODE REVIEW

File: ajax/libs/jszip-utils/package.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    "url": "https://github.com/Stuk/jszip-utils.git"
+  },
+  "keywords": [
+    "JSZip",
+    "ajax",
+    "cross browser",
+    "IE",
+    "Internet Explorer"
+  ],
+  "license": "MIT or GPLv3",
```
Comment (x09326) [inline ajax/libs/jszip-utils/package.json]: I'm not sure whether it is okay.
@PeterDaveHello please help me.

Comment (pvnr0082t) [inline ajax/libs/jszip-utils/package.json]: I think it should use licenses array. I'll modify it.

**Comment by pvnr0082t** [comment]
@x09326 I have modified the license to licenses array. Please help me review again, thank you.

**Comment by x09326** [comment]
@PeterDaveHello 
I think this PR is okay, please help me review it, thanks.

**Comment by pvnr0082t** [comment]
@x09326 I have rebased on the latest master branch. Please help me review again, thank you.

File: ajax/libs/jszip-utils/package.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  "keywords": [
+    "JSZip",
+    "ajax",
+    "cross browser",
+    "IE",
+    "Internet Explorer"
+  ],
+  "licenses": [
+    "MIT",
+    "GPLv3"
```
Comment (PeterDaveHello) [inline ajax/libs/jszip-utils/package.json]: Wrong.

**Comment by pvnr0082t** [comment]
@x09326 I have updated licenses field. Please help me review again, thank you.

**Comment by x09326** [comment]
@PeterDaveHello 
I think this PR is okay, please help me review it, thanks.

**Pull request merged**